### PR TITLE
[Navigation timing] Make sure unloadEvent is only exposed when all redirects pass TAO

### PR DESCRIPTION
**Pull request by yoavweiss**

### CODE REVIEW

**Comment by yoavweiss** [comment]
These are tests for https://github.com/w3c/navigation-timing/pull/103

**Comment by yoavweiss** [comment]
@toddreifsteck thanks for merging w3c/navigation-timing#103
Can you review the tests as well?

File: navigation-timing/nav2_test_redirect_chain_xserver_partial_opt_in.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        <title>Navigation Timing 2 WPT</title>
+        <link rel="author" title="Google" href="http://www.google.com/" />
+        <link rel="help" href="http://www.w3.org/TR/navigation-timing-2/#sec-PerformanceNavigationTiming"/>
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+        <script src="/common/utils.js"></script>
+        <script>
+
+            function verifyTimingEventOrder(eventOrder, timingEntry) {
+                for (var i = 0; i < eventOrder.length - 1; i++) {
```
Comment (npm1) [inline navigation-timing/nav2_test_redirect_chain_xserver_partial_opt_in.html]: nit here and throughout: use let or const instead of var

**Review by npm1 — APPROVED** [review]
LGTM % nit

File: navigation-timing/nav2_test_redirect_chain_xserver_partial_opt_in.html (hunk 1 above)
Comment (yoavweiss) [reply navigation-timing/nav2_test_redirect_chain_xserver_partial_opt_in.html]: Yeah, this was copied from an existing test. I'll change it

**Comment by yoavweiss** [comment]
@npm1  - Thanks for reviewing! :)

**Pull request merged**